client/dashboard/style.scss

```diff
@@ -33,6 +33,7 @@
 	margin: 0 auto;
 	width: 84px;
 	padding: $gap-large;
+	margin-top: -$gap-large;
```

Comment:
you can add this to the `margin` declaration a few lines above. Or, remove the top padding.

Reply:
Absolutely, good catch!